diff --git a/node_modules/@codemirror/view/dist/index.cjs b/node_modules/@codemirror/view/dist/index.cjs
index 2e8ca40..f4072e3 100644
--- a/node_modules/@codemirror/view/dist/index.cjs
+++ b/node_modules/@codemirror/view/dist/index.cjs
@@ -3809,6 +3809,12 @@ function posAtCoords(view, coords, precise, scanY) {
     let line = view.docView.lineAt(block.from, 2);
     if (!line || line.length != block.length)
         line = view.docView.lineAt(block.from, -2);
+    if (!line) {
+        if (precise)
+            return null;
+        let pos = posAtCoordsImprecise(view, content, block, x, y);
+        return new PosAssoc(pos, pos == block.from ? 1 : -1);
+    }
     return new InlineCoordsScan(view, x, y, view.textDirectionAt(block.from)).scanTile(line, block.from);
 }
 class InlineCoordsScan {
@@ -3922,6 +3928,8 @@ class InlineCoordsScan {
         // on the y axis, move this.y into it, and retry the scan.
         if (!closestRect) {
             let side = above && (!below || (this.y - above.bottom < below.top - this.y)) ? above : below;
+            if (!side)
+                return { i: lo, after: false };
             this.y = (side.top + side.bottom) / 2;
             return this.scan(positions, getRects, true);
         }
diff --git a/node_modules/@codemirror/view/dist/index.js b/node_modules/@codemirror/view/dist/index.js
index cfb4459..536cbde 100644
--- a/node_modules/@codemirror/view/dist/index.js
+++ b/node_modules/@codemirror/view/dist/index.js
@@ -3805,6 +3805,12 @@ function posAtCoords(view, coords, precise, scanY) {
     let line = view.docView.lineAt(block.from, 2);
     if (!line || line.length != block.length)
         line = view.docView.lineAt(block.from, -2);
+    if (!line) {
+        if (precise)
+            return null;
+        let pos = posAtCoordsImprecise(view, content, block, x, y);
+        return new PosAssoc(pos, pos == block.from ? 1 : -1);
+    }
     return new InlineCoordsScan(view, x, y, view.textDirectionAt(block.from)).scanTile(line, block.from);
 }
 class InlineCoordsScan {
@@ -3918,6 +3924,8 @@ class InlineCoordsScan {
         // on the y axis, move this.y into it, and retry the scan.
         if (!closestRect) {
             let side = above && (!below || (this.y - above.bottom < below.top - this.y)) ? above : below;
+            if (!side)
+                return { i: lo, after: false };
             this.y = (side.top + side.bottom) / 2;
             return this.scan(positions, getRects, true);
         }
